packages/adp-flp-config-sub-generator/src/app/index.ts

```diff
@@ -0,0 +1,197 @@
+import type { Manifest } from '@sap-ux/project-access';
+import type { FlpConfigOptions } from './types';
+import type { Question } from 'inquirer';
+import type { AbapServiceProvider } from '@sap-ux/axios-extension';
+import type { AbapTarget } from '@sap-ux/system-access';
+import { getSystemSelectionQuestions } from '@sap-ux/odata-service-inquirer';
+import Generator from 'yeoman-generator';
+import path, { join } from 'path';
+import {
+    ManifestService,
+    getVariant,
+    getAdpConfig,
+    getInboundsFromManifest,
+    getRegistrationIdFromManifest,
+    isCFEnvironment,
+    generateInboundConfig,
+    type InternalInboundNavigation
+} from '@sap-ux/adp-tooling';
+import { ToolsLogger } from '@sap-ux/logger';
+import { EventName } from '../telemetryEvents';
+import { getPrompts, type FLPConfigAnswers } from '@sap-ux/flp-config-inquirer';
+import { AppWizard, Prompts, MessageType } from '@sap-devx/yeoman-ui-types';
+import { TelemetryHelper, sendTelemetry } from '@sap-ux/fiori-generator-shared';
+import { isInternalFeaturesSettingEnabled } from '@sap-ux/feature-toggle';
+import { FileName } from '@sap-ux/project-access';
+import { isAppStudio } from '@sap-ux/btp-utils';
+import { SystemService, BackendSystemKey } from '@sap-ux/store';
+import AdpFlpConfigLogger from '../utils/logger';
+import { t } from '../utils/i18n';
+
+/**
+ * Generator for adding a FLP configuration to an Adaptation Project.
+ *
+ * @extends Generator
+ */
+export default class extends Generator {
+    setPromptsCallback: (fn: object) => void;
+    prompts: Prompts;
+    launchFlpConfigAsSubGenerator: boolean;
+    appWizard: AppWizard;
+    manifest: Manifest;
+    projectRootPath: string = '';
+    answers: FLPConfigAnswers;
+    logger: ToolsLogger;
+
+    /**
+     * Creates an instance of the generator.
+     *
+     * @param {string | string[]} args - The arguments passed to the generator.
+     * @param {FlpConfigOptions} opts - The options for the generator.
+     */
+    constructor(args: string | string[], opts: FlpConfigOptions) {
+        // Force the generator to overwrite existing files without additional prompting
+        opts.force = true;
+        super(args, opts);
+        this.appWizard = opts.appWizard ?? AppWizard.create(opts);
+        this.launchFlpConfigAsSubGenerator = Boolean(opts.launchFlpConfigAsSubGenerator);
```

Comment:
You can just use shorthand "!!"
```suggestion
        this.launchFlpConfigAsSubGenerator = !!opts.launchFlpConfigAsSubGenerator;
```

Some ideas (optional):
It would be good to shorten this variable for readability, flp is contextually understood. 
For example: `launchAsSubGen`. 

And then we could add a comment on what this variable is for above where all of the properties are defined.

Reply:
Fixed in: 575d3a5e06ff953c7b3ce6d1886e72fc4a7190b9